mern-ecommerce/routes/auth.js

```diff
@@ -11,6 +11,7 @@ const { userSignupValidator } = require('../validator');
 
 router.post('/signup', userSignupValidator, signup);
 router.post('/signin', signin);
-router.get('/signout', signout);
+router.get('/signout', signouts);
```

Comment:
The function `signouts` is not defined in the provided context. It seems like it should be `signout` instead of `signouts`.